### ISSUE TITLE
Message after clicking "sign in" disappears on macOS.

### ISSUE THREAD

**Issue by SzymonJaworskiRF** [ZbayApp/zbay#271]
Message: "Starting up. This may take a while..." appears only for a very short period of time. Then it disappears.

Platform: macOS

Steps to reproduce: 
* Quit zBay
* Open zBay
* Click sign in

Info: This happened to me when I was testing "quit zbay and reopened in really fast". On windows everything looks fine.

![Zrzut ekranu 2020-07-8 o 02 02 05](https://user-images.githubusercontent.com/67907892/86900394-237efc80-c0c0-11ea-990c-880f1cc47d9b.png)

**Comment by holmesworcester** [comment]
was zbay able to connect in this case?

**Comment by SzymonJaworskiRF** [comment]
Yes, I've tried it about 5 times and every time the connection was successful.
